ansible/roles/npm/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+-   name: install node sources
+    shell: curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+
+-   name: install nodejs
+    apt:
+        name: nodejs
+        state: present
+
+-   name: install npm
+    apt:
+        name: npm
+        state: present
+
+-   name: install yarn
```

Comment:
is this the recommended way of installing yarn right now? I thought their apt repo would be the better choice?

Reply:
Jay :D its not the best way, i´ll have a look for it